etc/config/c++.amazon.properties

```diff
@@ -4039,6 +4039,14 @@ libs.pugixml.staticliblink=pugixml
 libs.pugixml.versions.1114.version=1.11.4
 libs.pugixml.versions.1114.path=/opt/compiler-explorer/libs/pugixml/v1.11.4/src/
 
+libs.pybind11.name=Pybind11
+libs.pybind11.url=https://github.com/pybind/pybind11
+libs.pybind11.versions=trunk:2_11_1
+libs.pybind11.version.trunk.version=trunk
```

Comment:
sorry I missed this one: this and lines below should be prefixed with `libs.pybind11.versions.`
(versions with an s)
So in this case `libs.pybind11.versions.trunk.version=trunk`

Reply:
That's what happens when I try to contribute while tired.

Fixed.